### PR TITLE
Display the class and module names when the file has no methods.

### PR DESCRIPTION
**Pull request by Onumis**
Proposal fix for #59

### CODE REVIEW

File: test/lib/rubycritic/core/analysed_modules_collection_test.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
       let(:paths) { "test/samples/" }
 
       it "recursively registers all files" do
-        subject.count.must_equal 13
+        subject.count.must_equal 14
```
Comment (Onumis) [inline test/lib/rubycritic/core/analysed_modules_collection_test.rb]: This is a pita... we have to increment this if we add new sample files to test with...

What about we change to: `subject.count.must_equal Dir['test/samples/**/*.rb'].count`?

Comment (troessner) [inline test/lib/rubycritic/core/analysed_modules_collection_test.rb]: Good idea!

**Comment by troessner** [comment]
One comment and then lgtm!

**Comment by Onumis** [comment]
@troessner updated!

**Comment by troessner** [comment]
Nice! Merged ;)

**Pull request merged**